port/zos390/omrsignal_context_ceehdlr.h

```diff
@@ -43,12 +43,12 @@ typedef struct {
 	uint32_t portLibrarySignalType;
 	void *handlerAddress;
 	void *handlerAddress2;
-} J9LEConditionInfo;
+} OMRLEConditionInfo;
 
```

Comment:
Does this need a typedef similar to J9ZOSLEConditionHandlerRecord?

Reply:
This struct is only used in OMR. Internal J9 zOS builds didn't complain about missing `J9LEConditionInfo`.

Reply:
I don't think the **J9** structs especially the **ZOS** ones are used by projects other than J9. If you are worried about non-J9 projects, we can add **defines** for the renamed structs. Or we can take some risk and delay the introduction of **defines** if we find other projects are impacted by the renaming.

Reply:
Also, there is no public function which exposes `OMRLEConditionInfo` and `omr_31bit_mch` to an OMR consumer so it should be safe to exclude the `defines` during renaming. The same applies to `J9ZOSLEConditionHandlerRecord`, which is an internal OMR structure and not exposed via OMR public functions. OpenJ9 shouldn't use `J9ZOSLEConditionHandlerRecord` directly. Do we guarantee support for private structures used within OMR? If not, we can remove the `define` for `J9ZOSLEConditionHandlerRecord` from this pull request, and break J9 and force it to amend their implementation.